CONTRIBUTING.md

```diff
@@ -119,7 +119,7 @@ Turns into
 
 ### GT4Py stencils
 
-We interface to `gt4py.cartesian.gtscript.stencil` through ndsl.dsl.stencil, specifically the FrozenStencil, that allows us to minimize runtime overhead in calling stencils.
+We interface to `ndsl.dsl.gt4py.stencil` through ndsl.dsl.stencil, specifically the FrozenStencil, that allows us to minimize runtime overhead in calling stencils.
```

Comment:
If we're changing this line maybe we could say "we interface to gt4py.cartesian stencils" since this redundancy seems a bit awkward